sdk/sdkservices/scheduler.go

```diff
@@ -0,0 +1,13 @@
+package sdkservices
+
+import (
+	"context"
+
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+type Scheduler interface {
+	Create(ctx context.Context, scheduleID string, schedule string, triggerID sdktypes.TriggerID) error
```

Comment:
is scheduleID an actual ID, free text? if it's an ID, it should be in sdktypes. This can be done in a separate PR.

Reply:
scheduleID is text, it's an ID we are passing to temporal to name schedule. It's not an sdkType and not intended to be

Reply:
name can change, you need to use trigger id here.

Reply:
done

Reply:
i think you didn't push the latest changes - i still see scheduleID.

Reply:
I pushed. We don't save `scheduleID` anymore in trigger data, since scheduleID == triggerID as you wanted (e.g. not prepending a trigger name to it)

we need to pass `ScheduleID` is to create an normal API, same way we are doing for every other object. There are 3 functions in the interface - Create, Update and Delete. Very similar to all other our interfaces. 
- Update and Delete should receive `ScheduleID` (the same way any connection function receives `connectionID` or trigger functions receives `triggerID` etc..).
- Create could receive either `Schedule` object and return scheduleID (if we want to add Schedule to sdktypes). Right now I see no point in doing so, so `Create` also receives `scheduleID`.

Reply:
I pushed. We don't save `scheduleID` anymore in trigger data, since scheduleID == triggerID as you wanted (e.g. not prepending a trigger name to it)

we need to pass `ScheduleID` is to create an normal API, same way we are doing for every other object. There are 3 functions in the interface - Create, Update and Delete. Very similar to all other our interfaces. 
- Update and Delete should receive `ScheduleID` (the same way any connection function receives `connectionID` or trigger functions receives `triggerID` etc..).
- Create could receive either `Schedule` object and return scheduleID (if we want to add Schedule to sdktypes). Right now I see no point in doing so, so `Create` also receives `scheduleID`.